scrapscript.py

```diff
@@ -1254,6 +1255,91 @@ def bencode(obj: object) -> bytes:
     raise NotImplementedError(f"bencode not implemented for {type(obj)}")
 
 
+class JSCompiler:
```

Comment:
🙏 you know what I'm thinking

Reply:
imports are our friends

Reply:
This is just a prototype before doing it in scrap